clients/java/src/main/java/io/camunda/zeebe/client/impl/ZeebeClientImpl.java

```diff
@@ -390,4 +393,9 @@ public ThrowErrorCommandStep1 newThrowErrorCommand(final long jobKey) {
   public ThrowErrorCommandStep1 newThrowErrorCommand(final ActivatedJob job) {
     return newThrowErrorCommand(job.getKey());
   }
+
+  @Override
+  public ActivateJobsCommandStep1 newActivateJobsCommand() {
+    return jobClient.newActivateJobsCommand();
+  }
```

Comment:
:upside_down_face: Is there any reason why you moved this method at the bottom of the class? Wouldn't it make more sense to place this method next to the `newStreamJobsCommand` since they both have a similar role?